tensorboard/webapp/metrics/effects/metrics_effects_test.ts

```diff
@@ -89,8 +89,52 @@ describe('metrics effects', () => {
       selectors.getMetricsTooltipSort,
       TooltipSort.ALPHABETICAL
     );
+
+    overrideTagMetadata();
+    overrideRunToEid();
   });
 
+  function overrideTagMetadata() {
+    store.overrideSelector(selectors.getMetricsTagMetadata, {
+      scalars: {
+        tagDescriptions: {} as any,
+        tagToRuns: {
+          tagA: ['run1'],
```

Comment:
This is not the run name structure I see in practice.

![image](https://github.com/tensorflow/tensorboard/assets/17152369/71ac2274-b624-405c-acc6-ef009dd7f070)